PIL/Image.py

```diff
@@ -1122,6 +1186,7 @@ def filter(self, filter):
         return merge(self.mode, ims)
 
     def getbands(self):
+        # type: () -> Tuple[Any]
```

Comment:
I'm not sure there's a way to specify a 'char' in the type (ie. length==1), but we can certainly add a type alias for `Band = Text`, which documents it nicely?

Aliases of the same underlying type are interchangeable (eg. `Band` and `Mode`), but we could go beyond this and make them separate types.